Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -759,7 +867,7 @@ void GameListCtrl::OnKeyPress(wxListEvent& event)
   static int lastKey = 0, sLoop = 0;
   int Loop = 0;
 
-  for (int i = 0; i < (int)m_ISOFiles.size(); i++)
+  for (int i = 0; i < (int)m_shown_files.size(); i++)
```

Comment:
I tried not to touch _too_ much of existing code. Really the loop this function is in should be rewritten completely, but it's not part of this PR, and I'd rather not get lost on putting more effort into WX than necessary.